include/igl/copyleft/comiso/nrosy.cpp

```diff
@@ -257,7 +257,8 @@ void igl::copyleft::comiso::NRosyField::prepareSystemMatrix(const int N)
   // System sizes: A (count_t + count_p) x (count_t + count_p)
   //               b (count_t + count_p)
 
-  b = Eigen::VectorXd::Zero(count_t + count_p);
+  b.resize(count_t + count_p);
```

Comment:
How is that better that assigning a `VectorXd::Zero`? :p I'm pretty sure the former is matrix expression and doesn't do any memory allocation until it actually gets assigned to `b`.

Reply:
You may be right. I will have a look at this. Thanks for this comment.